src/placeholder.ts

```diff
@@ -57,6 +54,7 @@ export class Placeholder<A> extends Behavior<A> {
     if (this.source !== undefined) {
       this.source.addListener(this.node, t);
       if (isBehavior(this.source)) {
+        this.pull(t);
```

Comment:
Why is this pull there? As far as I can see all the tests still pass if I comment it out.